src/kbc_modules/eaa_kbc/mod.rs

```diff
@@ -171,12 +177,11 @@ impl EAAKbc {
         }
     }
 
-    fn kbs_get_resource(&mut self, description: &str) -> Result<Vec<u8>> {
-        let desc: ResourceDescription = serde_json::from_str::<ResourceDescription>(description)?;
-
-        let command = format!("Get {}", desc.name.as_str());
-        let request = GetResourceRequest::new(&command, desc.optional.clone());
-        let resource_info = self.kbs_get_resource_info(desc.name.as_str())?;
+    fn kbs_get_resource(&mut self, rid: &ResourceId) -> Result<Vec<u8>> {
+        let resource_path = rid.resource_path();
+        let command = format!("Get {resource_path}");
```

Comment:
If the command is modified here, `Verdictd` needs to be modified in to match it.

Reply:
Yes, related issue is https://github.com/inclavare-containers/verdictd/issues/29 and we could handle later